core/sbt-cloudflow/src/main/scala/cloudflow/sbt/CloudflowBasePlugin.scala

```diff
@@ -94,17 +94,17 @@ object CloudflowBasePlugin extends AutoPlugin {
     buildAndPublishImage := Def.task {
           val _ = (checkUncommittedChanges.value, verifyDockerRegistry.value)
           Def.task {
-            val streamletDescriptors = streamletDescriptorsInProject.value
-            val imageId: ImageId     = dockerBuildAndPush.value
-            val log                  = streams.value.log
-            val imagesPushed         = (imageNames in docker).value
+            val streamletDescriptors                 = streamletDescriptorsInProject.value
+            val imageId: Map[ImageName, ImageDigest] = dockerBuildAndPush.value
+            val log                                  = streams.value.log
+            val imagesPushed                         = (imageNames in docker).value
             if (imagesPushed.size > 1) throw TooManyImagesBuilt
```

Comment:
I wonder if this even makes sense..

Reply:
When would this happen? (I know, not part of this PR, but still, don't really get it here)